doctrine/registration_form.rst

```diff
@@ -3,166 +3,114 @@
    single: Form; Simple Registration Form
    single: Security; Simple Registration Form
 
-How to Implement a Simple Registration Form
-===========================================
+How to Implement a Registration Form
+====================================
 
-Creating a registration form works the same as creating any form. You configure
-the form to update some ``User`` model object (a Doctrine entity in this
+Creating a registration form works the same as creating any form: you create a
```

Comment:
This intro is misleading. It reads as: "creating a registration form is exactly the same as creating a normal form". The reader may wonder: then why do you publish a special article for it?

In my opinion the intro should say something like this:

* The basics of creating a registration form are the same as any normal form. After all, you are creating/edition an object with it (a user).
* However, since this is related to security, you need to do something else to integrate that into your app's security. This article explains it all.

Reply:
Very nice suggestion